packages/notification-center/src/components/notification-center-web-component/notification-center-component.types.ts

```diff
@@ -0,0 +1,50 @@
+import type { INovuProviderProps } from '../novu-provider';
+import type { IPopoverNotificationCenterProps } from '../popover-notification-center';
+import type { ColorScheme } from '../../shared/config/colors';
+import { NotificationCenterStyles } from '../../store/styles';
+
+type SelectedProviderProps = Pick<
+  INovuProviderProps,
+  | 'onLoad'
+  | 'stores'
+  | 'backendUrl'
+  | 'socketUrl'
+  | 'subscriberId'
+  | 'applicationIdentifier'
+  | 'subscriberHash'
+  | 'i18n'
+>;
+type NovuProviderProps = SelectedProviderProps & {
+  // Angular/Vue props
+  sessionLoaded?: INovuProviderProps['onLoad'];
+};
+
+// Angular/Vue props
+interface PopoverAdditionalProps {
+  notificationClicked?: IPopoverNotificationCenterProps['onNotificationClick'];
+  unseenCountChanged?: IPopoverNotificationCenterProps['onUnseenCountChanged'];
+  actionClicked?: IPopoverNotificationCenterProps['onActionClick'];
+  tabClicked?: IPopoverNotificationCenterProps['onTabClick'];
+}
```

Comment:
The thing is that all these props are defined on the `NotificationCenterWebComponent` which is basically the proxy to the React component - `NotificationCenterComponent`, so the props interface definition should leave in the `notification-center` package and should be defined through the one interface `NotificationCenterComponentProps`.

These are called "additional" props because the original ones are: `onNotificationClick, onUnseenCountChanged, onActionClick, onTabClick` and in `Angular/Vue` world you can't use prop names that start with the `on` prefix because of some security considerations, that's why these siblings were created.
Both wrapper libraries do use these props.

Maybe, I've clarified more about why it's like that.